criu/arch/s390/crtools.c

```diff
@@ -86,8 +86,9 @@ static void print_core_gs_cb(CoreEntry *core)
 		pr_debug("       No GS_CB\n");
 		return;
 	}
-	for (i = 0; i < 4; i++)
+	for (i = 0; i < 4; i++){
```

Comment:
Please, don't add irrelevant changes into a patch. Also criu's coding style doesn't welcome braces around single line blocks.

Reply:
I added these braces to the code because when I tried to build criu after adding binary logging, I had gotten errors like: else without a previous if. This error was fixed after adding braces to it. 
I understand that this goes against the coding practice and have found a better way of doing it.